mpxplay/in_file.h

```diff
@@ -233,7 +233,7 @@ typedef struct mpxplay_filehand_low_data_s{
 
 //low level file handling (normal-file, cdw, ftp-file-source)
 typedef struct mpxplay_filehand_low_func_s{
- int   (*check) (char *filename);     // true (1) or false (0) (check filename (ie:filename begins with "http:\\"), use these functions if 'true')
+ int   (*check) (char *filename);     // true (1) or false (0) (check filename (ie:filename begins with "http://"), use these functions if 'true')
```

Comment:
This change is oddly unrelated to the rest of the PR, and could probably be done as a separate cleanup PR?